### PR TITLE
Setup easy setup

### PR DESCRIPTION
**Pull request by sir-marc**

### CODE REVIEW

File: benchmark/README.md [hunk 1]
```diff
@@ -4,4 +4,8 @@ The benchmark expects `../demo` project to be running.
 
 The benchmark will automatically run a number of samples, to customize the amount of samples, pass `--iterations=n`.
 
-Interpreting the data is currently wip but the logs give an indication of what is already being tracked.
\ No newline at end of file
+Interpreting the data is currently wip but the logs give an indication of what is already being tracked.
+
+## Dev
+Run `yarn setup` to get up and running and then `yarn start`
+**Note**: The `../demo` project needs to be running as well.
```
Comment (StuckiSimon) [inline benchmark/README.md:11]: That's already written 3 lines above

Comment (sir-marc) [reply benchmark/README.md:11]: I know, just a friendly reminder ^^

File: demo/package.json [hunk 2]
```diff
@@ -7,7 +7,8 @@
     "start": "snowpack dev",
     "build": "snowpack build",
     "lode-cli": "node ../cli/index.js",
-    "serve": "serve ./build -p 8080"
+    "serve": "serve ./build -p 8080",
+    "setup": "yarn install && yarn lode-cli && yarn upgrade lode-three"
```
Comment (StuckiSimon) [inline demo/package.json:11]: IMO this setup should also call setup in the cli?

Comment (sir-marc) [reply demo/package.json:11]: Even though that would lead to multiple calls when run from the parent directory?

Comment (StuckiSimon) [reply demo/package.json:11]: It's fine for now as we don't have a nice monorepo setup, but this is kind of unintuitive.
There would be no drawback when calling the setup of the cli 😄

**Review by StuckiSimon — APPROVED** [review]
looks good 👍

**Pull request merged**